Package.swift

```diff
@@ -4,134 +4,144 @@ import PackageDescription
+                "FilterJSON",
+                "FilterSets",
             ]
-        )
-    ],
-    dependencies: [
+        ),
+        .library(
+            name: "Recording",
+            targets: ["Recording"]
+        ),
     ],
     targets: [
+        //MARK: - Recording
         .target(
-            name: "KSCrash/Installations",
-            path: "Source/KSCrash/Installations",
-            publicHeadersPath: ".",
-            cxxSettings: [
-                .headerSearchPath("../Recording"),
-                .headerSearchPath("../Recording/Monitors"),
-                .headerSearchPath("../Recording/Tools"),
-                .headerSearchPath("../Reporting/Filters"),
-                .headerSearchPath("../Reporting/Sinks"),
-                .headerSearchPath("../Reporting/Tools"),
+            name: "Recording",
+            dependencies: [
+                "RecordingTools",
+                "FilterBase",
+            ],
+            cSettings: [.headerSearchPath("Monitors")]
+        ),
+        .target(
+            name: "RecordingTools",
+            dependencies: [
+                "CommonTools",
+                "KSCrashSwift",
+                "KSCrashLLVM",
             ]
         ),
+        .target(name: "CommonTools"),
+        //MARK: - Peporting
         .target(
-            name: "KSCrash/Recording",
-            path: "Source/KSCrash/Recording",
-            exclude: [
-                "Monitors",
-                "Tools"
-            ],
-            resources: [
-                .copy("PrivacyInfo.xcprivacy")
-            ],
-            publicHeadersPath: ".",
-            cxxSettings: [
-                .headerSearchPath("Tools"),
-                .headerSearchPath("Monitors"),
-                .headerSearchPath("../Reporting/Filters")
+            name: "ReportingTools",
+            dependencies: ["CommonTools"],
+            linkerSettings: [
+                .linkedFramework("SystemConfiguration", .when(platforms: [.iOS, .tvOS, .macOS]))
             ]
         ),
         .target(
-            name: "KSCrash/Recording/Monitors",
-            path: "Source/KSCrash/Recording/Monitors",
-            publicHeadersPath: ".",
-            cxxSettings: [
-                .define("GCC_ENABLE_CPP_EXCEPTIONS", to: "YES"),
-                .headerSearchPath(".."),
-                .headerSearchPath("../Tools"),
-                .headerSearchPath("../../Reporting/Filters")
+            name: "ReportingSinks",
+            dependencies: [
+                "CommonTools",
+                "ReportingTools",
+                "FilterBase",
+                "FilterAlert",
+                "FilterAppleFmt",
+                "FilterBasic",
+                "FilterStringify",
+                "FilterGZip",
+                "FilterJSON",
+            ],
+            linkerSettings: [
+                .linkedFramework("MessageUI", .when(platforms: [.iOS]))
             ]
         ),
+        // MARK: - Filters
```

Comment:
Filters modules are created according to the [.podspec](https://github.com/kstenerud/KSCrash/blob/master/KSCrash.podspec). But now I'm thinking, maybe there's no need for all these targets and 1 target with all filters would be enough.